rosidl_generator_py/cmake/find_valid_typesupport.cmake

```diff
@@ -0,0 +1,34 @@
+# Copyright 2016 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+macro(rosidl_generator_py_find_valid_typesupport TYPESUPPORT)
+  set(${TYPESUPPORT} "")
+  foreach(_extension IN LISTS AMENT_EXTENSIONS_rosidl_generate_interfaces)
```

Comment:
yeah, actually, I think this will actually disable the tests even when there is a valid typesupport impl... I think `AMENT_EXTENSIONS_rosidl_generate_interfaces` comes from `rosidl_generator_py_extras`, which calls `ament_register_extension`... Fixing it now